Razor/Scripts/Outlands.cs

```diff
@@ -19,20 +19,17 @@
 #endregion
 
 using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Windows.Forms;
 using Assistant.Core;
-using Assistant.HotKeys;
 using Assistant.Scripts.Engine;
-using Assistant.Scripts.Helpers;
 
 namespace Assistant.Scripts
 {
     public static class Outlands
     {
+        // Take all list of BuffIcon to string
+        private static readonly string[] buffNames = Enum.GetNames(typeof(BuffIcon));
```

Comment:
Make this a Dictionary<string, BuffIcon> where the key is the lower case string name. Then you can very quickly look up the buff icon below.

Reply:
introduce
- we need to fill up every usable buff that we want to catch via script